test/getUser.spec.js

```diff
@@ -0,0 +1,40 @@
+/* eslint-env jest */
+
+const { user } = require('../lib');
+const { generateRandomString, ConnectionFactory } = require('./setup-database');
+
+const { role } = user;
+
+describe('user.get', () => {
+  let conn;
+
+  beforeEach(() => {
```

Comment:
Let's stop this pattern. Just do `const conn = ConnectionFactory()` we don't do anything to mutate `conn` that would require a new one each test.